src/interpreter/vm/VM.cpp

```diff
@@ -17,21 +17,24 @@ VM::VM(unsigned char *bytecode) :
     // Do nothing.
 }
 
-VM::~VM() {
-
-}
+VM::~VM() {}
 
 
 int VM::run() {
-    //DurianObject a, b, v;
     while(true) {
         unsigned char opcode = nextBytecode();
         DURIAN_DEBUG_LOG("%x\n", opcode);
         DurianObject a, b;
-        int32_t jumpLen;
+        int32_t jump_len; // Jump length
```

Comment:
Why was this changed from camel case to underscores? I thought we were doing camel case?